src/qibo/noise_model.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
+from qibo import gates, models
+from qibo.noise import DepolarizingError, NoiseModel, ThermalRelaxationError
+
+
+def noise_model(Circuit, params):
```

Comment:
I don't understand why you are passing as first argument the `Circuit` class definition instead of a simple class instance.